bindings/python/google_benchmark/benchmark.cc

```diff
@@ -161,12 +153,12 @@ NB_MODULE(_benchmark, m) {
       .def_prop_ro("error_occurred", &State::error_occurred)
       .def("set_iteration_time", &State::SetIterationTime)
       .def_prop_rw("bytes_processed", &State::bytes_processed,
-                    &State::SetBytesProcessed)
+                   &State::SetBytesProcessed)
       .def_prop_rw("complexity_n", &State::complexity_length_n,
-                    &State::SetComplexityN)
+                   &State::SetComplexityN)
       .def_prop_rw("items_processed", &State::items_processed,
-                    &State::SetItemsProcessed)
-      .def("set_label", (void (State::*)(const char*)) & State::SetLabel)
+                   &State::SetItemsProcessed)
+      .def("set_label", (void(State::*)(const char*)) & State::SetLabel)
```

Comment:
This would need to bechanged to accept a `const std::string&` too, or does it work with implicit conversion?

Reply:
hm. interesting that the bindings CI bot didn't fail.